cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * (C) Copyright IBM Corp. 2020, 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.engine.api.service;
+
+import java.util.Date;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+
+import com.ibm.cohort.engine.api.service.model.EvaluateMeasureResults;
+import com.ibm.cohort.engine.api.service.model.EvaluateMeasuresStatus;
+import com.ibm.cohort.engine.api.service.model.MeasuresEvaluation;
+import com.ibm.watson.common.service.base.ServiceBaseConstants;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.SwaggerDefinition;
+import io.swagger.annotations.Tag;
+
+/**
+ * Cohort REST service
+ *
+ *
+ */
+@Path(CohortEngineRestConstants.SERVICE_MAJOR_VERSION)
+@Api(value = "cohort")
+@SwaggerDefinition(tags={@Tag(name = "measures", description = "IBM Cohort Engine Measure Evaluation")})
+public class CohortEngineRestHandler {
+	//private static final Logger logger = LoggerFactory.getLogger(CohortEngineRestHandler.class.getName());
```

Comment:
I know this isn't really doing anything yet, but is there a reason to comment out the logger?

Reply:
Just to remove an unused variable warning in eclipse. I'm happy to uncomment it as it will be used when these REST apis are fleshed out.

Reply:
I was wrong. Having it uncommented causes a PMD violation and fails the build <violation beginline="47" endline="47" begincolumn="37" endcolumn="42" rule="UnusedPrivateField" ruleset="Best Practices" package="com.ibm.cohort.engine.api.service" class="CohortEngineRestHandler" variable="logger" externalInfoUrl="https://pmd.github.io/pmd-6.21.0/pmd_rules_java_bestpractices.html#unusedprivatefield" priority="3">
Avoid unused private fields such as 'logger'.
</violation>

Reply:
Interesting, thanks